supplier_management_site/tests/test.py

```diff
@@ -1,13 +1,23 @@
-from django.test import SimpleTestCase, RequestFactory
-from django.utils.translation import activate, ugettext_lazy as _
 from parameterized import parameterized
-from .views import home
 
+from django.core import mail
+from django.test import TestCase, RequestFactory
+from django.utils.translation import activate, ugettext_lazy as _
```

Comment:
Here it could be more readable if you do a multiline import

Reply:
solved in next PR